### PR TITLE
Adds the common-sign-language-0002 model

### PR DESCRIPTION
**Pull request by evgeny-izutov**

### CODE REVIEW

File: data/dataset_definitions.yml [hunk 1]
```diff
@@ -981,6 +981,16 @@ datasets:
...
     annotation: jester_action_recognition.pickle
 
+  - name: common_sign_language_gestures
+    data_source: common_sign_language_gestures/global_crops
+    annotation_conversion:
+      converter: continuous_clip_action_recognition
+      annotation_file: common_sign_language_gestures/test.txt
+      data_dir: common_sign_language_gestures/global_crops
+      out_fps: 15
+      clip_length: 8
```
Comment (Anna-Mironova) [inline data/dataset_definitions.yml:991]:
```suggestion
      clip_length: 8
      img_prefix: ''
```

Comment (evgeny-izutov) [reply data/dataset_definitions.yml:991]: Done

**Comment by eaidova** [comment]
@evgeny-izutov could you please provide classes.json for using model in demo?
Usually we store such files in this directory https://github.com/openvinotoolkit/open_model_zoo/tree/develop/data/dataset_classes
and also mention it in model description

**Comment by evgeny-izutov** [comment]
@eaidova, I've added model-related info into the gesture-recognition demo description.

File: demos/gesture_recognition_demo/python/README.md [hunk 2]
```diff
@@ -102,7 +103,11 @@ python gesture_recognition_demo.py \
 
 The demo starts in person tracking mode and to switch it in the action recognition mode you should press `0-9` button with appropriate detection ID (the number in top-left of each bounding box). If frame contains only one person, they will be chosen automatically. After that you can switch back to tracking mode by pressing space button.
 
-An example of file with class names can be found within the OMZ directory at `<omz_dir>/data/dataset_classes/msasl100.json`.
+An example of file with class names can be found within the OMZ directory:
+
+* MS-ASL-100: `<omz_dir>/data/dataset_classes/msasl100.json`
+* Jester-27: `<omz_dir>/data/dataset_classes/jester27.json`
+* Common-Sign-Language-12: `<omz_dir>/data/dataset_classes/common_sign_language.json`
```
Comment (eaidova) [inline demos/gesture_recognition_demo/python/README.md:110]: I think it makes sense to add number of classes to file name for avoid conflicts in the future if will be added models with different number of supported classes

Comment (evgeny-izutov) [reply demos/gesture_recognition_demo/python/README.md]: Done

**Review by eaidova — APPROVED** [review]

**Review by Anna-Mironova — APPROVED** [review]

**Pull request merged**